src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/common/stateful_init_graph.cpp

```diff
@@ -248,12 +248,15 @@ class InitGraphStatefulDiffPrimitiveModel : public InitGraphStatefulModelBase {
     }
 
     void check_init_graph_node() override {
-        CheckNumberOfNodesWithType(compiledModel, "Convert", function->is_dynamic() ? 1 : 0);
+        CheckNumberOfNodesWithType(compiledModel, "Convert", 0);
     }
 
     ov::Shape get_state_shape(size_t i) override {
         return inputShapes[0].second[i];
     }
+
+private:
+    const ov::element::Type exePrc = ElementType::i8;
```

Comment:
Suggest setting inference_precision hint to `fp32` rather than build the model in `int8`, to keep the inference precision consistent across HW platforms. Also please set the model and state precision to `fp32`.

Reply:
ok, reverted i8, and added inference_precision_hint f32.